study-builder/studies/pancan/prequal.conf

```diff
@@ -310,20 +310,42 @@
               "shownExpr": null
             },
             {
-              "question": ${_includes.diagnosed_primary_cancers} {
-                "stableId": "DIAGNOSED_PRIMARY_CANCERS"
+              "question": ${_includes.diagnosed_primary_cancer_list} {
+                "stableId": "DIAGNOSED_PRIMARY_CANCER_LIST"
```

Comment:
Please pull out identifiers to `subs.conf`.

Reply:
I sort of like question stableIds not in substitutions but refactored any way.